utilities/tools/Test-ModuleLocally.ps1

```diff
@@ -279,9 +269,9 @@ function Test-ModuleLocally {
             } catch {
                 Write-Error $_
             } finally {
-                # Restore parameter files
-                # -----------------------
-                if (($ValidationTest -or $DeploymentTest) -and $ValidateOrDeployParameters) {
+                # Restore test files
+                # ------------------
+                if (($ValidationTest -or $DeploymentTest) -and $ValidateOrDeployParameters -and $moduleTestFiles) {
```

Comment:
Isn't $moduleTestFiles implicit? As in it always has a value?

Reply:
Fair point. Removed